Running `make test` to run unit tests fails when `gtk4` is installed locally

### Bug description

Attempting to run `make test` to invoke the Unit Test fails due to the `progress_box.py importing `Gtk` and `Pango` from the `gi.repository` module without specifying a Gtk version.

```
(.venv) [user@host /mnt/DataDrive/workspace/lutris]$ make test
rm tests/fixtures/pga.db -f
nose2
2026-03-23 18:44:04,373: Command 'vulkaninfo' not found on your system
/mnt/DataDrive/workspace/lutris/lutris/gui/widgets/progress_box.py:3: PyGIWarning: Gtk was imported without specifying a version first. Use gi.require_version('Gtk', '4.0') before import to ensure that the right version gets loaded.
  from gi.repository import Gtk, Pango
...
======================================================================
ERROR: runners.test_yaml (nose2.loader.ModuleImportFailure.runners.test_yaml)
----------------------------------------------------------------------
ImportError: Failed to import test module: runners.test_yaml
Traceback (most recent call last):
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/importer.py", line 137, in create_module
    introspection_module = get_introspection_module(namespace)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/module.py", line 247, in get_introspection_module
    module = IntrospectionModule(namespace, version)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/module.py", line 107, in __init__
    repository.require(namespace, version)
    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
gi.RepositoryError: Requiring namespace 'Gtk' version '3.0', but '4.0' is already loaded

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/plugins/loader/discovery.py", line 195, in _find_tests_in_file
    module = util.module_from_name(module_name)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/util.py", line 73, in module_from_name
    __import__(name)
    ~~~~~~~~~~^^^^^^
  File "/mnt/DataDrive/workspace/lutris/tests/runners/test_yaml.py", line 3, in <module>
    from lutris.runners import yaml
  File "/mnt/DataDrive/workspace/lutris/lutris/runners/yaml.py", line 7, in <module>
    from lutris.runners.model import ModelRunner
  File "/mnt/DataDrive/workspace/lutris/lutris/runners/model.py", line 8, in <module>
    from lutris.runners.runner import Runner
  File "/mnt/DataDrive/workspace/lutris/lutris/runners/runner.py", line 10, in <module>
    from lutris import runtime, settings
  File "/mnt/DataDrive/workspace/lutris/lutris/runtime.py", line 25, in <module>
    from lutris.util.wine.d3d_extras import D3DExtrasManager
  File "/mnt/DataDrive/workspace/lutris/lutris/util/wine/d3d_extras.py", line 1, in <module>
    from lutris.util.wine.dll_manager import DLLManager
  File "/mnt/DataDrive/workspace/lutris/lutris/util/wine/dll_manager.py", line 16, in <module>
    from lutris.util.wine.prefix import WinePrefixManager
  File "/mnt/DataDrive/workspace/lutris/lutris/util/wine/prefix.py", line 7, in <module>
    from lutris.util.display import DISPLAY_MANAGER
  File "/mnt/DataDrive/workspace/lutris/lutris/util/display.py", line 15, in <module>
    from gi.repository import GnomeDesktop  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/importer.py", line 139, in create_module
    raise ImportError(e) from e
ImportError: Requiring namespace 'Gtk' version '3.0', but '4.0' is already loaded


======================================================================
ERROR: runners.test_runners (nose2.loader.ModuleImportFailure.runners.test_runners)
----------------------------------------------------------------------
ImportError: Failed to import test module: runners.test_runners
Traceback (most recent call last):
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/plugins/loader/discovery.py", line 195, in _find_tests_in_file
    module = util.module_from_name(module_name)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/util.py", line 73, in module_from_name
    __import__(name)
    ~~~~~~~~~~^^^^^^
  File "/mnt/DataDrive/workspace/lutris/tests/runners/test_runners.py", line 4, in <module>
    from test_pga import DatabaseTester
  File "/mnt/DataDrive/workspace/lutris/tests/test_pga.py", line 8, in <module>
    from lutris.util.test_config import setup_test_environment
  File "/mnt/DataDrive/workspace/lutris/lutris/util/test_config.py", line 5, in <module>
    gi.require_version("Gdk", "3.0")
    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/__init__.py", line 139, in require_version
    raise ValueError(
        f"Namespace {namespace} is already loaded with version {loaded_version}"
    )
ValueError: Namespace Gdk is already loaded with version 4.0


======================================================================
ERROR: runners.test_json (nose2.loader.ModuleImportFailure.runners.test_json)
----------------------------------------------------------------------
ImportError: Failed to import test module: runners.test_json
Traceback (most recent call last):
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/importer.py", line 137, in create_module
    introspection_module = get_introspection_module(namespace)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/module.py", line 247, in get_introspection_module
    module = IntrospectionModule(namespace, version)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/module.py", line 107, in __init__
    repository.require(namespace, version)
    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
gi.RepositoryError: Requiring namespace 'Gtk' version '3.0', but '4.0' is already loaded

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/plugins/loader/discovery.py", line 195, in _find_tests_in_file
    module = util.module_from_name(module_name)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/util.py", line 73, in module_from_name
    __import__(name)
    ~~~~~~~~~~^^^^^^
  File "/mnt/DataDrive/workspace/lutris/tests/runners/test_json.py", line 3, in <module>
    from lutris.runners import json
  File "/mnt/DataDrive/workspace/lutris/lutris/runners/json.py", line 8, in <module>
    from lutris.runners.model import ModelRunner
  File "/mnt/DataDrive/workspace/lutris/lutris/runners/model.py", line 8, in <module>
    from lutris.runners.runner import Runner
  File "/mnt/DataDrive/workspace/lutris/lutris/runners/runner.py", line 10, in <module>
    from lutris import runtime, settings
  File "/mnt/DataDrive/workspace/lutris/lutris/runtime.py", line 25, in <module>
    from lutris.util.wine.d3d_extras import D3DExtrasManager
  File "/mnt/DataDrive/workspace/lutris/lutris/util/wine/d3d_extras.py", line 1, in <module>
    from lutris.util.wine.dll_manager import DLLManager
  File "/mnt/DataDrive/workspace/lutris/lutris/util/wine/dll_manager.py", line 16, in <module>
    from lutris.util.wine.prefix import WinePrefixManager
  File "/mnt/DataDrive/workspace/lutris/lutris/util/wine/prefix.py", line 7, in <module>
    from lutris.util.display import DISPLAY_MANAGER
  File "/mnt/DataDrive/workspace/lutris/lutris/util/display.py", line 15, in <module>
    from gi.repository import GnomeDesktop  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/importer.py", line 139, in create_module
    raise ImportError(e) from e
ImportError: Requiring namespace 'Gtk' version '3.0', but '4.0' is already loaded


======================================================================
ERROR: runners.test_pcsx2 (nose2.loader.ModuleImportFailure.runners.test_pcsx2)
----------------------------------------------------------------------
ImportError: Failed to import test module: runners.test_pcsx2
Traceback (most recent call last):
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/importer.py", line 137, in create_module
    introspection_module = get_introspection_module(namespace)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/module.py", line 247, in get_introspection_module
    module = IntrospectionModule(namespace, version)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/module.py", line 107, in __init__
    repository.require(namespace, version)
    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
gi.RepositoryError: Requiring namespace 'Gtk' version '3.0', but '4.0' is already loaded

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/plugins/loader/discovery.py", line 195, in _find_tests_in_file
    module = util.module_from_name(module_name)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/util.py", line 73, in module_from_name
    __import__(name)
    ~~~~~~~~~~^^^^^^
  File "/mnt/DataDrive/workspace/lutris/tests/runners/test_pcsx2.py", line 5, in <module>
    from lutris.runners.pcsx2 import pcsx2
  File "/mnt/DataDrive/workspace/lutris/lutris/runners/pcsx2.py", line 7, in <module>
    from lutris.runners.runner import Runner
  File "/mnt/DataDrive/workspace/lutris/lutris/runners/runner.py", line 10, in <module>
    from lutris import runtime, settings
  File "/mnt/DataDrive/workspace/lutris/lutris/runtime.py", line 25, in <module>
    from lutris.util.wine.d3d_extras import D3DExtrasManager
  File "/mnt/DataDrive/workspace/lutris/lutris/util/wine/d3d_extras.py", line 1, in <module>
    from lutris.util.wine.dll_manager import DLLManager
  File "/mnt/DataDrive/workspace/lutris/lutris/util/wine/dll_manager.py", line 16, in <module>
    from lutris.util.wine.prefix import WinePrefixManager
  File "/mnt/DataDrive/workspace/lutris/lutris/util/wine/prefix.py", line 7, in <module>
    from lutris.util.display import DISPLAY_MANAGER
  File "/mnt/DataDrive/workspace/lutris/lutris/util/display.py", line 15, in <module>
    from gi.repository import GnomeDesktop  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/importer.py", line 139, in create_module
    raise ImportError(e) from e
ImportError: Requiring namespace 'Gtk' version '3.0', but '4.0' is already loaded


======================================================================
ERROR: runners.test_snes9x (nose2.loader.ModuleImportFailure.runners.test_snes9x)
----------------------------------------------------------------------
ImportError: Failed to import test module: runners.test_snes9x
Traceback (most recent call last):
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/importer.py", line 137, in create_module
    introspection_module = get_introspection_module(namespace)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/module.py", line 247, in get_introspection_module
    module = IntrospectionModule(namespace, version)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/module.py", line 107, in __init__
    repository.require(namespace, version)
    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
gi.RepositoryError: Requiring namespace 'Gtk' version '3.0', but '4.0' is already loaded

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/plugins/loader/discovery.py", line 195, in _find_tests_in_file
    module = util.module_from_name(module_name)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/util.py", line 73, in module_from_name
    __import__(name)
    ~~~~~~~~~~^^^^^^
  File "/mnt/DataDrive/workspace/lutris/tests/runners/test_snes9x.py", line 4, in <module>
    from lutris.runners.snes9x import snes9x
  File "/mnt/DataDrive/workspace/lutris/lutris/runners/snes9x.py", line 10, in <module>
    from lutris.runners.runner import Runner
  File "/mnt/DataDrive/workspace/lutris/lutris/runners/runner.py", line 10, in <module>
    from lutris import runtime, settings
  File "/mnt/DataDrive/workspace/lutris/lutris/runtime.py", line 25, in <module>
    from lutris.util.wine.d3d_extras import D3DExtrasManager
  File "/mnt/DataDrive/workspace/lutris/lutris/util/wine/d3d_extras.py", line 1, in <module>
    from lutris.util.wine.dll_manager import DLLManager
  File "/mnt/DataDrive/workspace/lutris/lutris/util/wine/dll_manager.py", line 16, in <module>
    from lutris.util.wine.prefix import WinePrefixManager
  File "/mnt/DataDrive/workspace/lutris/lutris/util/wine/prefix.py", line 7, in <module>
    from lutris.util.display import DISPLAY_MANAGER
  File "/mnt/DataDrive/workspace/lutris/lutris/util/display.py", line 15, in <module>
    from gi.repository import GnomeDesktop  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/importer.py", line 139, in create_module
    raise ImportError(e) from e
ImportError: Requiring namespace 'Gtk' version '3.0', but '4.0' is already loaded


======================================================================
ERROR: runners.test_model (nose2.loader.ModuleImportFailure.runners.test_model)
----------------------------------------------------------------------
ImportError: Failed to import test module: runners.test_model
Traceback (most recent call last):
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/importer.py", line 137, in create_module
    introspection_module = get_introspection_module(namespace)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/module.py", line 247, in get_introspection_module
    module = IntrospectionModule(namespace, version)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/module.py", line 107, in __init__
    repository.require(namespace, version)
    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
gi.RepositoryError: Requiring namespace 'Gtk' version '3.0', but '4.0' is already loaded

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/plugins/loader/discovery.py", line 195, in _find_tests_in_file
    module = util.module_from_name(module_name)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/util.py", line 73, in module_from_name
    __import__(name)
    ~~~~~~~~~~^^^^^^
  File "/mnt/DataDrive/workspace/lutris/tests/runners/test_model.py", line 7, in <module>
    from lutris.runners.model import (
    ...<2 lines>...
    )
  File "/mnt/DataDrive/workspace/lutris/lutris/runners/model.py", line 8, in <module>
    from lutris.runners.runner import Runner
  File "/mnt/DataDrive/workspace/lutris/lutris/runners/runner.py", line 10, in <module>
    from lutris import runtime, settings
  File "/mnt/DataDrive/workspace/lutris/lutris/runtime.py", line 25, in <module>
    from lutris.util.wine.d3d_extras import D3DExtrasManager
  File "/mnt/DataDrive/workspace/lutris/lutris/util/wine/d3d_extras.py", line 1, in <module>
    from lutris.util.wine.dll_manager import DLLManager
  File "/mnt/DataDrive/workspace/lutris/lutris/util/wine/dll_manager.py", line 16, in <module>
    from lutris.util.wine.prefix import WinePrefixManager
  File "/mnt/DataDrive/workspace/lutris/lutris/util/wine/prefix.py", line 7, in <module>
    from lutris.util.display import DISPLAY_MANAGER
  File "/mnt/DataDrive/workspace/lutris/lutris/util/display.py", line 15, in <module>
    from gi.repository import GnomeDesktop  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/importer.py", line 139, in create_module
    raise ImportError(e) from e
ImportError: Requiring namespace 'Gtk' version '3.0', but '4.0' is already loaded


======================================================================
ERROR: runners.test_model_validator (nose2.loader.ModuleImportFailure.runners.test_model_validator)
----------------------------------------------------------------------
ImportError: Failed to import test module: runners.test_model_validator
Traceback (most recent call last):
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/importer.py", line 137, in create_module
    introspection_module = get_introspection_module(namespace)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/module.py", line 247, in get_introspection_module
    module = IntrospectionModule(namespace, version)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/module.py", line 107, in __init__
    repository.require(namespace, version)
    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
gi.RepositoryError: Requiring namespace 'Gtk' version '3.0', but '4.0' is already loaded

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/plugins/loader/discovery.py", line 195, in _find_tests_in_file
    module = util.module_from_name(module_name)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/util.py", line 73, in module_from_name
    __import__(name)
    ~~~~~~~~~~^^^^^^
  File "/mnt/DataDrive/workspace/lutris/tests/runners/test_model_validator.py", line 5, in <module>
    from lutris.runners.model import (
        ModelRunner,
    )
  File "/mnt/DataDrive/workspace/lutris/lutris/runners/model.py", line 8, in <module>
    from lutris.runners.runner import Runner
  File "/mnt/DataDrive/workspace/lutris/lutris/runners/runner.py", line 10, in <module>
    from lutris import runtime, settings
  File "/mnt/DataDrive/workspace/lutris/lutris/runtime.py", line 25, in <module>
    from lutris.util.wine.d3d_extras import D3DExtrasManager
  File "/mnt/DataDrive/workspace/lutris/lutris/util/wine/d3d_extras.py", line 1, in <module>
    from lutris.util.wine.dll_manager import DLLManager
  File "/mnt/DataDrive/workspace/lutris/lutris/util/wine/dll_manager.py", line 16, in <module>
    from lutris.util.wine.prefix import WinePrefixManager
  File "/mnt/DataDrive/workspace/lutris/lutris/util/wine/prefix.py", line 7, in <module>
    from lutris.util.display import DISPLAY_MANAGER
  File "/mnt/DataDrive/workspace/lutris/lutris/util/display.py", line 15, in <module>
    from gi.repository import GnomeDesktop  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/importer.py", line 139, in create_module
    raise ImportError(e) from e
ImportError: Requiring namespace 'Gtk' version '3.0', but '4.0' is already loaded


======================================================================
ERROR: runners.test_vita3k (nose2.loader.ModuleImportFailure.runners.test_vita3k)
----------------------------------------------------------------------
ImportError: Failed to import test module: runners.test_vita3k
Traceback (most recent call last):
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/importer.py", line 137, in create_module
    introspection_module = get_introspection_module(namespace)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/module.py", line 247, in get_introspection_module
    module = IntrospectionModule(namespace, version)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/module.py", line 107, in __init__
    repository.require(namespace, version)
    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
gi.RepositoryError: Requiring namespace 'Gtk' version '3.0', but '4.0' is already loaded

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/plugins/loader/discovery.py", line 195, in _find_tests_in_file
    module = util.module_from_name(module_name)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/util.py", line 73, in module_from_name
    __import__(name)
    ~~~~~~~~~~^^^^^^
  File "/mnt/DataDrive/workspace/lutris/tests/runners/test_vita3k.py", line 4, in <module>
    from lutris.runners.vita3k import MissingVitaTitleIDError, vita3k
  File "/mnt/DataDrive/workspace/lutris/lutris/runners/vita3k.py", line 5, in <module>
    from lutris.runners.json import JSON_RUNNER_DIRS, JsonRunner
  File "/mnt/DataDrive/workspace/lutris/lutris/runners/json.py", line 8, in <module>
    from lutris.runners.model import ModelRunner
  File "/mnt/DataDrive/workspace/lutris/lutris/runners/model.py", line 8, in <module>
    from lutris.runners.runner import Runner
  File "/mnt/DataDrive/workspace/lutris/lutris/runners/runner.py", line 10, in <module>
    from lutris import runtime, settings
  File "/mnt/DataDrive/workspace/lutris/lutris/runtime.py", line 25, in <module>
    from lutris.util.wine.d3d_extras import D3DExtrasManager
  File "/mnt/DataDrive/workspace/lutris/lutris/util/wine/d3d_extras.py", line 1, in <module>
    from lutris.util.wine.dll_manager import DLLManager
  File "/mnt/DataDrive/workspace/lutris/lutris/util/wine/dll_manager.py", line 16, in <module>
    from lutris.util.wine.prefix import WinePrefixManager
  File "/mnt/DataDrive/workspace/lutris/lutris/util/wine/prefix.py", line 7, in <module>
    from lutris.util.display import DISPLAY_MANAGER
  File "/mnt/DataDrive/workspace/lutris/lutris/util/display.py", line 15, in <module>
    from gi.repository import GnomeDesktop  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/importer.py", line 139, in create_module
    raise ImportError(e) from e
ImportError: Requiring namespace 'Gtk' version '3.0', but '4.0' is already loaded


======================================================================
ERROR: runners.test_scummvm (nose2.loader.ModuleImportFailure.runners.test_scummvm)
----------------------------------------------------------------------
ImportError: Failed to import test module: runners.test_scummvm
Traceback (most recent call last):
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/importer.py", line 137, in create_module
    introspection_module = get_introspection_module(namespace)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/module.py", line 247, in get_introspection_module
    module = IntrospectionModule(namespace, version)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/module.py", line 107, in __init__
    repository.require(namespace, version)
    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
gi.RepositoryError: Requiring namespace 'Gtk' version '3.0', but '4.0' is already loaded

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/plugins/loader/discovery.py", line 195, in _find_tests_in_file
    module = util.module_from_name(module_name)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/util.py", line 73, in module_from_name
    __import__(name)
    ~~~~~~~~~~^^^^^^
  File "/mnt/DataDrive/workspace/lutris/tests/runners/test_scummvm.py", line 4, in <module>
    from lutris.config import LutrisConfig
  File "/mnt/DataDrive/workspace/lutris/lutris/config.py", line 8, in <module>
    from lutris import settings, sysoptions
  File "/mnt/DataDrive/workspace/lutris/lutris/sysoptions.py", line 11, in <module>
    from lutris.util.display import DISPLAY_MANAGER, SCREEN_SAVER_INHIBITOR, is_compositing_enabled, is_display_x11
  File "/mnt/DataDrive/workspace/lutris/lutris/util/display.py", line 15, in <module>
    from gi.repository import GnomeDesktop  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/importer.py", line 139, in create_module
    raise ImportError(e) from e
ImportError: Requiring namespace 'Gtk' version '3.0', but '4.0' is already loaded


======================================================================
ERROR: test_installer (nose2.loader.ModuleImportFailure.test_installer)
----------------------------------------------------------------------
ImportError: Failed to import test module: test_installer
Traceback (most recent call last):
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/plugins/loader/discovery.py", line 195, in _find_tests_in_file
    module = util.module_from_name(module_name)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/util.py", line 73, in module_from_name
    __import__(name)
    ~~~~~~~~~~^^^^^^
  File "/mnt/DataDrive/workspace/lutris/tests/test_installer.py", line 3, in <module>
    from lutris.installer.errors import ScriptingError
  File "/mnt/DataDrive/workspace/lutris/lutris/installer/errors.py", line 7, in <module>
    from lutris.gui.dialogs import ErrorDialog
  File "/mnt/DataDrive/workspace/lutris/lutris/gui/dialogs/__init__.py", line 14, in <module>
    gi.require_version("Gdk", "3.0")
    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/__init__.py", line 139, in require_version
    raise ValueError(
        f"Namespace {namespace} is already loaded with version {loaded_version}"
    )
ValueError: Namespace Gdk is already loaded with version 4.0


======================================================================
ERROR: test_wine (nose2.loader.ModuleImportFailure.test_wine)
----------------------------------------------------------------------
ImportError: Failed to import test module: test_wine
Traceback (most recent call last):
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/importer.py", line 137, in create_module
    introspection_module = get_introspection_module(namespace)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/module.py", line 247, in get_introspection_module
    module = IntrospectionModule(namespace, version)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/module.py", line 107, in __init__
    repository.require(namespace, version)
    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
gi.RepositoryError: Requiring namespace 'Gtk' version '3.0', but '4.0' is already loaded

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/plugins/loader/discovery.py", line 195, in _find_tests_in_file
    module = util.module_from_name(module_name)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/util.py", line 73, in module_from_name
    __import__(name)
    ~~~~~~~~~~^^^^^^
  File "/mnt/DataDrive/workspace/lutris/tests/test_wine.py", line 3, in <module>
    from lutris.runners import wine
  File "/mnt/DataDrive/workspace/lutris/lutris/runners/wine.py", line 9, in <module>
    from lutris import runtime, settings
  File "/mnt/DataDrive/workspace/lutris/lutris/runtime.py", line 25, in <module>
    from lutris.util.wine.d3d_extras import D3DExtrasManager
  File "/mnt/DataDrive/workspace/lutris/lutris/util/wine/d3d_extras.py", line 1, in <module>
    from lutris.util.wine.dll_manager import DLLManager
  File "/mnt/DataDrive/workspace/lutris/lutris/util/wine/dll_manager.py", line 16, in <module>
    from lutris.util.wine.prefix import WinePrefixManager
  File "/mnt/DataDrive/workspace/lutris/lutris/util/wine/prefix.py", line 7, in <module>
    from lutris.util.display import DISPLAY_MANAGER
  File "/mnt/DataDrive/workspace/lutris/lutris/util/display.py", line 15, in <module>
    from gi.repository import GnomeDesktop  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/importer.py", line 139, in create_module
    raise ImportError(e) from e
ImportError: Requiring namespace 'Gtk' version '3.0', but '4.0' is already loaded


======================================================================
ERROR: test_dialogs (nose2.loader.ModuleImportFailure.test_dialogs)
----------------------------------------------------------------------
ImportError: Failed to import test module: test_dialogs
Traceback (most recent call last):
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/plugins/loader/discovery.py", line 195, in _find_tests_in_file
    module = util.module_from_name(module_name)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/util.py", line 73, in module_from_name
    __import__(name)
    ~~~~~~~~~~^^^^^^
  File "/mnt/DataDrive/workspace/lutris/tests/test_dialogs.py", line 4, in <module>
    from lutris.gui.application import LutrisApplication
  File "/mnt/DataDrive/workspace/lutris/lutris/gui/application.py", line 31, in <module>
    gi.require_version("Gdk", "3.0")
    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/__init__.py", line 139, in require_version
    raise ValueError(
        f"Namespace {namespace} is already loaded with version {loaded_version}"
    )
ValueError: Namespace Gdk is already loaded with version 4.0


======================================================================
ERROR: test_pga (nose2.loader.ModuleImportFailure.test_pga)
----------------------------------------------------------------------
ImportError: Failed to import test module: test_pga
Traceback (most recent call last):
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/plugins/loader/discovery.py", line 195, in _find_tests_in_file
    module = util.module_from_name(module_name)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/util.py", line 73, in module_from_name
    __import__(name)
    ~~~~~~~~~~^^^^^^
  File "/mnt/DataDrive/workspace/lutris/tests/test_pga.py", line 8, in <module>
    from lutris.util.test_config import setup_test_environment
  File "/mnt/DataDrive/workspace/lutris/lutris/util/test_config.py", line 5, in <module>
    gi.require_version("Gdk", "3.0")
    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/__init__.py", line 139, in require_version
    raise ValueError(
        f"Namespace {namespace} is already loaded with version {loaded_version}"
    )
ValueError: Namespace Gdk is already loaded with version 4.0


======================================================================
ERROR: lutris.util.test_config (nose2.loader.ModuleImportFailure.lutris.util.test_config)
----------------------------------------------------------------------
ImportError: Failed to import test module: lutris.util.test_config
Traceback (most recent call last):
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/plugins/loader/discovery.py", line 195, in _find_tests_in_file
    module = util.module_from_name(module_name)
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/nose2/util.py", line 73, in module_from_name
    __import__(name)
    ~~~~~~~~~~^^^^^^
  File "/mnt/DataDrive/workspace/lutris/lutris/util/test_config.py", line 5, in <module>
    gi.require_version("Gdk", "3.0")
    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^
  File "/mnt/DataDrive/workspace/lutris/.venv/lib/python3.14/site-packages/gi/__init__.py", line 139, in require_version
    raise ValueError(
        f"Namespace {namespace} is already loaded with version {loaded_version}"
    )
ValueError: Namespace Gdk is already loaded with version 4.0


----------------------------------------------------------------------
Ran 202 tests in 1.195s

FAILED (errors=14)
make: *** [Makefile:60: test] Error 1
```

relates #4688

### Workaorund
I am able to workaround the issue by updating the [progress_box.py](https://github.com/lutris/lutris/blob/master/lutris/gui/widgets/progress_box.py#L3) file to specify an explicit requirment on Gtk 3.0
```python
from gi import require_version

require_version("Gtk", "3.0")
```

### How to Reproduce

Steps to reproduce the behavior:
1. Make sure tha gtk4 is installed on the system.
  a. If using a package manager such as  pacman this can be validated by running the following command
```bash
[user@ua ~]$ pacman -Qs gtk4
local/gtk4 1:4.20.3-1
    GObject-based multi-platform GUI toolkit
```
2.  Invoke the `make test` command from the root of the lutris installation

### Expected behavior

That Unit Test are invocable when Gtk4 is installed on the contributors' system.

### Log output

```shell
This occurs when running unit test via the CLI
```

### System Information

```shell
Vulkan support: YES
Esync support: YES
Fsync support: YES
Wine installed: YES
Gamescope: YES
Mangohud: NO
Gamemode: YES
Steam: YES
In Flatpak: NO
[System]
OS: Arch Linux rolling n/a
Arch: x86_64
Kernel: 6.19.9-arch1-1
Lutris Version: 0.5.23
Python Version: 3.14.3 (main, Feb 13 2026, 15:31:44) [GCC 15.2.1 20260209]
Desktop: X-Cinnamon
Display Server: x11
[CPU]
Vendor: AuthenticAMD
Model: AMD Ryzen 9 9950X3D 16-Core Processor
Physical cores: 16
Logical cores: 32
[Memory]
RAM: 91.9 GB
Swap: 0.0 GB
[Graphics]
Vendor: AMD
OpenGL Renderer: AMD Radeon RX 7900 XTX (radeonsi, navi31, ACO, DRM 3.64, 6.19.9-arch1-1)
OpenGL Version: 4.6 (Compatibility Profile) Mesa 26.0.3-arch1.1
OpenGL Core: 4.6 (Core Profile) Mesa 26.0.3-arch1.1
OpenGL ES: OpenGL ES 3.2 Mesa 26.0.3-arch1.1
Vulkan Version: 1.4.341
Vulkan Drivers: AMD Ryzen 9 9950X3D 16-Core Processor (RADV RAPHAEL_MENDOCINO) (1.4.335), AMD Radeon RX 7900 XTX (RADV NAVI31) (1.4.335)
```

### Media (optional)

_No response_

--- a/lutris/gui/__init__.py
+++ b/lutris/gui/__init__.py
@@ -1,1 +1,10 @@
 """Lutris GUI package"""
+
+import gi
+
+gi.require_version("Gtk", "3.0")
+gi.require_version("Gdk", "3.0")
+gi.require_version("GnomeDesktop", "3.0")
+gi.require_version("Pango", "1.0")
+gi.require_version("PangoCairo", "1.0")
+gi.require_version("WebKit2", "4.1")

